package.json

```diff
@@ -69,6 +69,7 @@
     "webpack-dev-server-without-h2": "^2.11.8",
     "webpack-stats-plugin": "^0.1.5",
     "tabbable": "~5.2.0",
-    "sass": "~1.60.0"
+    "sass": "~1.60.0",
+    "@adobe/css-tools": "~4.2.0"
```

Comment:
Nit: I prefer things to remain sorted. We already have `@` dependencies on top.

Reply:
Updated..